src/containers/FilmFrontpage/AllMoviesAlphabetically.tsx

```diff
@@ -9,106 +9,58 @@
 import { useMemo } from "react";
 import { useTranslation } from "react-i18next";
 import { gql } from "@apollo/client";
-import styled from "@emotion/styled";
-import { breakpoints, mq, spacing, colors } from "@ndla/core";
-import { Image } from "@ndla/primitives";
+import { Heading, Text, Image, Skeleton } from "@ndla/primitives";
 import { SafeLink } from "@ndla/safelink";
-import { Heading, Text } from "@ndla/typography";
+import { HStack, styled } from "@ndla/styled-system/jsx";
+import { OneColumn } from "@ndla/ui";
 import { movieResourceTypes } from "./resourceTypes";
 import { GQLAllMoviesQuery, GQLAllMoviesQueryVariables } from "../../graphqlTypes";
 import { useGraphQuery } from "../../util/runQueries";
 
-const IMAGE_WIDTH = 143;
-
-const LetterHeading = styled(Heading)`
-  color: ${colors.white};
-  padding-left: ${spacing.nsmall};
-  padding-bottom: ${spacing.small};
-  border-bottom: 1px solid ${colors.brand.greyDark};
-`;
-
-const StyledWrapper = styled.section`
-  width: 652px;
-  max-width: 100%;
-  margin: ${spacing.large} auto;
-  padding: 0 ${spacing.normal};
-  display: flex;
-  flex-direction: column;
-  gap: ${spacing.normal};
-`;
-
-const MovieTextWrapper = styled.div`
-  ${mq.range({ from: breakpoints.tablet })} {
-    padding: ${spacing.xsmall} ${spacing.xsmall} 0 0;
-  }
-  flex: 1;
-`;
-
-const MovieImage = styled(Image)`
-  width: 104px;
-  height: 80px;
-  ${mq.range({ from: breakpoints.tablet })} {
-    width: ${IMAGE_WIDTH}px;
-    height: 90px;
-  }
-`;
-
-const MovieDescription = styled(Text)`
-  color: ${colors.brand.greyLighter};
-  margin: 0;
-  overflow: hidden;
-  text-overflow: ellipsis;
-  display: -webkit-box;
-  -webkit-box-orient: vertical;
-  -webkit-line-clamp: 2;
-`;
-
-const StyledSafeLink = styled(SafeLink)`
-  color: ${colors.black};
-  box-shadow: none;
-  display: flex;
-  gap: ${spacing.small};
-  &:hover,
-  &:focus {
-    [data-title] {
-      text-decoration: underline;
-    }
-  }
-`;
+const LetterHeading = styled(Heading, {
+  base: {
+    borderBottom: "1px solid",
+    borderColor: "stroke.subtle",
+  },
+});
+
+const MovieTextWrapper = styled("div", {
+  base: { flex: 1 },
+});
+
+const MovieImage = styled(Image, {
+  base: {
+    width: "surface.3xsmall",
+    tablet: {
+      width: "surface.3xsmall",
+    },
+  },
+});
+
+const StyledSafeLink = styled(SafeLink, {
+  base: {
+    display: "flex",
+    gap: "small",
+    minHeight: "surface.4xsmall",
+    overflow: "hidden",
+    "&:hover,&:active,&:focus-within": {
+      "& [data-title]": {
+        textDecoration: "underline",
+      },
+    },
```

Comment:
La oss heller fjerne underline på hover. MAO: flytt underline til "vanlig". Dropp active of focus-within. Vi har en focusring av en grunn 😄